src/Components/CreateImageWizard/formComponents/ReviewStep.js

```diff
@@ -98,6 +98,7 @@ const ReviewStep = () => {
                         }
                     </List>
                 </Tab>
+                {getState()?.values?.release.includes('rhel') === 'Red Hat Enterprise Linux (RHEL) 8' &&
```

Comment:
This will always evaluate to False. [.includes()](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/includes) returns a boolean. So testing if it equals 'Red Hat Enterprise Linux (RHEL) 8' will always return false.